ir/type.def

```diff
@@ -300,6 +300,16 @@ class Type_Struct : Type_StructLike {
 /// and some information about their types.
 class Type_UnknownStruct : Type_StructLike {
 #nodbprint
+    bool defaultInitializer = false;
```

Comment:
Let's call this "incomplete" rather than "defaultInitializer".
It is the type of an expression containing a default initializer (for now).